test/hotspot/jtreg/compiler/loopopts/superword/TestAlignVectorFuzzer.java

```diff
@@ -30,7 +30,7 @@
  * @key randomness
  * @run main/bootclasspath/othervm -XX:+IgnoreUnrecognizedVMOptions
  *                                 -XX:LoopUnrollLimit=250
- *                                 -XX:CompileCommand=printcompilation,compiler.loopopts.superword.TestAlignVectorFuzzer::*
+ *                                 -XX:CompileCommand=compileonly,compiler.loopopts.superword.TestAlignVectorFuzzer::*
  *                                 compiler.loopopts.superword.TestAlignVectorFuzzer
  */
```

Comment:
I think it would be good if we also had the same run but without the compileonly. That's what I meant by duplication ;)

Reply:
I added a separate run in a new commit.